packages/eas-cli/src/submissions/android/ServiceAccountSource.ts

```diff
@@ -44,6 +57,29 @@ async function handlePathSourceAsync(source: ServiceAccountPathSource): Promise<
   return source.path;
 }
 
+async function handleDetectSourceAsync(_source: ServiceAccountDetectSoruce): Promise<string> {
+  const projectDir = (await findProjectRootAsync()) ?? process.cwd();
+  const foundFilenames = await glob('**/*.json', {
+    cwd: projectDir,
+    ignore: ['app.json', 'package*.json', 'tsconfig.json', 'node_modules'],
+  });
+
+  const googleServiceFiles = await filterAsync(
+    foundFilenames.map(file => path.join(projectDir, file)),
+    fileIsGoogleServicesAsync
+  );
+
+  if (googleServiceFiles.length > 0) {
+    const detectedPath = googleServiceFiles[0];
+
+    if (await confirmDetectedPathAsync(detectedPath)) {
+      return detectedPath;
+    }
+  }
```

Comment:
Maybe if we've found multiple files and the user rejected the first one, we could propose another? Or maybe better we could show a select.

Reply:
I considered that. A few thoughts about that:
- I may be wrong, but I think that's a rare case when there's more than one google-services file.
- Proposing another file after first one is rejected, can be super annoying - a list-select is much better option here...
- but in the most common case (1 valid file), displaying a one-item list won't look nice, unless the last item would be "Provide path manually", which redirects to prompt for path

Reply:
We could display a select only if there are multiple choices.

Reply:
My comment was truncated lol 🤦 the last point was just about what you said: <=1 choice - current PR behavior; >1 - display the list.
But I have no idea which is a better UX - list only with 1 item or decide based on the number of items.

Reply:
i think it's better to only prompt when there are multiple items, because we can make the choice for them if there is just one item

Reply:
I've decided to go with the flow: _Ask for confirmation if single file; Display a list on multiple_

I was experimenting with different approaches discussed above and felt like this one is the best UX of them.
In typical case (1 file found) it's just one Enter/Return press (confirm defaults to _yes_) - still a huge improvement from prompting to type path each time, without losing flexibility.